bdb/attr.h

```diff
@@ -444,6 +444,7 @@ DEF_ATTR(AA_MIN_PERCENT, aa_min_percent, QUANTITY, 20,
          "Percent change above which we kick off analyze.")
 DEF_ATTR(AA_MIN_PERCENT_JITTER, aa_min_percent_jitter, QUANTITY, 300,
          "Additional jitter factor for determining percent change.")
+DEF_ATTR(AA_TESTING, aa_testing, BOOLEAN, 0, "turn AA TESTING on/off")
```

Comment:
I had this idea of keeping all the *debugging/testing* related tunables under "debug/test" namespace in the tunables sub-system (something like debug.aa_testing). That way, it will be easier to identify them in lrl files and elsewhere. Also, better description wouldn't hurt. :)